### PR TITLE
Update menor.js

### PR DESCRIPTION
**Pull request by Menor25**

### CODE REVIEW

**Comment by dhatGuy** [comment]
Working on it

**Review by dhatGuy — COMMENTED** [review]
You are outputting the result twice and also, change your 'i' to 'I'.

**Comment by Menor25** [comment]
so which of the output should i remove?

**Comment by Rubylena** [comment]
remove from function down

**Review by alxzndr2019 — APPROVED** [review]

File: scripts/menor.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+var fullName = "Theophilus Ajiri Menor"; 
+var hngId = "HNG-02809";
+var langauge = "javascript";
+var email = "ajirimenor@gmail.com";
 
-    var myOutput = "Hello World, i am " + fullName + " with HNGi7 ID " + hngId + " and email " + email + " using " + langauge + " for stage 2 task";
+var myOutput = "Hello World, i am " + fullName + " with HNGi7 ID " + hngId + " and email " + email + " using " + langauge + " for stage 2 task";
+
+console.log(myOutput);
+function menorOutput(){
```
Comment (simply-alliv) [inline scripts/menor.js]: There is no need for this function as you have used the myOutput variable in the previous line (line 9) to console.log your output.

Comment (Menor25) [reply scripts/menor.js]: that is my old script, i have removed the function, waiting for you to merge the new one. I am sending another update now with script name menor_update.js

Comment (Menor25) [reply scripts/menor.js]: This is the link of the one i corrected https://github.com/hng-teamfury/team-fury-1/blob/dev/scripts/menor.js

**Pull request merged**